servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/RepeatStrategies.java

```diff
@@ -261,7 +263,8 @@ private RepeatStrategies() {
             if (repeatCount > maxRepeats) {
                 return terminateRepeat();
             }
-            final long baseDelayNanos = min(maxDelayNanos, initialDelayNanos << min(maxInitialShift, repeatCount - 1));
+            final long baseDelayNanos = baseDelayNanos(initialDelayNanos, maxDelayNanos, maxInitialShift,
+                    repeatCount - 1);
```

Comment:
`repeatCount - 1` -> `repeatCount`

Reply:
yeah I noticed right after I pushed the change 😢 ... sending an update